### PR TITLE
Update javascript-7-oop.md

### PR DESCRIPTION
**Pull request by imskr**
Issue #594

### CODE REVIEW

**Comment by imskr** [comment]
I've added extra resources too. @alodahl Please review.

**Comment by imskr** [comment]
@Shwetha-Acharya Thanks for reviewing, I will remove it.

**Comment by imskr** [comment]
Ok got it. I will add those terms.

**Comment by imskr** [comment]
@alodahl Done.

File: javascript/javascript-7-oop.md [hunk 1]
```diff
@@ -321,8 +321,19 @@ The `Loan` constructor is using the `calculateDueDate()` method polymorphically.
...
-Work through [this Introduction to Classes on CodeAcademy](https://www.codecademy.com/courses/learn-javascript-classes/lessons/classes/exercises/introduction?course_redirect=introduction-to-javascript).
+* Work through [this interactive lesson on Advanced JavaScript Objects](https://www.codecademy.com/courses/introduction-to-javascript/lessons/advanced-objects/exercises/adv-intro?action=resume_content_item).
+
+* Work through [this Introduction to Classes on CodeAcademy](https://www.codecademy.com/courses/learn-javascript-
+classes/lessons/classes/exercises/introduction?course_redirect=introduction-to-javascript).
+
+* Find definition of following terms on your own:
+   * Namespace
+   * Hoisting
+   * Arity
```
Comment (Shwetha-Acharya) [inline javascript/javascript-7-oop.md]: Just curious, are you trying to establish the fact that the arity property which was used to return the number of arguments expected by the function, no longer exists and has been replaced by the Function.prototype.length property?? @alodahl What do you think, what is good to mention here, arity or Function.prototype.length?

Comment (Shwetha-Acharya) [reply javascript/javascript-7-oop.md]: I would say to replace arity by Function.prototype.length.

Comment (Shwetha-Acharya) [reply javascript/javascript-7-oop.md]: Anyways as a concept/definition it is well and good to explore about arity. But you should establish the fact specific to javascript. You can suggest to go through the link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/arity after mentioning Arity.

Comment (imskr) [reply javascript/javascript-7-oop.md]: Thank You for pointing out. Arity is now replaced by Function.prototype.length but Arity can also be used. I will add it too.

**Comment by alodahl** [comment]
Tip for future: describe specific changes in PR rather than * Update javascript-7-oop.md.  Great work, thank you =)

**Pull request merged**